ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -55,3 +55,8 @@ tests:
   openshift_installer:
     cluster_profile: gcp
     upgrade: true
+- as: windows-e2e-ovn
+  commands: ""
```

Comment:
What forced this to be added? You shouldn't have to

Reply:
it is a holdover from my ovn-e2e PR which probably got it from a doc or another workflow that is out of date. 

I removed it